example.js

```diff
@@ -82,21 +82,54 @@ function thermostats() {
           function() {
             return fritz.getTemperature(thermostat).then(function(temp) {
               temp = isNaN(temp) ? '-' : temp + "°C";
-              console.log("[" + thermostat + "] temp " + temp + "°C");
+              console.log("[" + thermostat + "] temp: " + temp);
             });
           },
           function() {
             return fritz.getTempTarget(thermostat).then(function(temp) {
               temp = isNaN(temp) ? '-' : temp + "°C";
-              console.log("[" + thermostat + "] target temp " + temp + "°C\n");
+              console.log("[" + thermostat + "] target temp: " + temp + "\n");
             });
-          }
+          }          
         ]);
       };
     }));
   });
 }
 
+// display bulb information
+function bulbs() {
+  return fritz.getBulbList().then(function(bulbs) {
+    console.log("Bulbs: " + bulbs + "\n");
+
+    return sequence(bulbs.map(function(bulb) {
```

Comment:
Where does this sequence function come from?

Reply:
I'm not sure whether I understand what you mean. 
getBulbList returns a list of bulb ains which is then used just the same way like the ain list is used for the switches or thermostats. The only difference is that the innermost sequence is a list with only one api function call. A list with one element is still a list ;-)
I used more than one function in the innermost list while testing the bulb , so the structure was quite handy.

Reply:
I think what I wanted to ask is what is `sequence`? Couldn't find the function anywhere- is that a js language construct that I have not seen before?

Reply:
Ah, ok. Well, it is defined in example.js line 18 to 24 :-)

Reply:
ahhh. wrong file ⭕. will take a look and merge asap